src/processors/chain/ProcessorChain.cpp

```diff
@@ -175,8 +175,20 @@ void ProcessorChain::processAudio (AudioBuffer<float>& buffer)
             inputBuffer.copyFrom (ch, 0, osBlock.getChannelPointer ((size_t) ch), osNumSamples);
     }
 
-    // run processing chain
+    // process standalone modulation ports
     bool outProcessed = false;
+    for (auto* processor : procs)
+    {
+        auto numImputs = processor->getNumInputConnections();
+
+        auto isModProcessor = processor->getProcessorType() == Modulation;
```

Comment:
Is this redundant? Like I'm thinking that if this is false, then `modOutputConnected` would also be false, so maybe we don't need to explicitly check this as well? Or maybe we should just move this check into `BaseProcessor::isOutputModulationPortConnected()`?

Reply:
Yeah it seems like we don't need it after doing some testing, good call. The only thing I can think of is checking if it's a modulation processor before looping through the connections to save doing that if it's not a modulation processor?